src/timedate/timedated.c

```diff
@@ -725,7 +725,7 @@ static int method_set_local_rtc(sd_bus_message *m, void *userdata, sd_bus_error
         if (r < 0)
                 return r;
 
-        if (lrtc == c->local_rtc)
+        if (lrtc == c->local_rtc && fix_system == 0)
```

Comment:
given fix_system is actually a bool (just one stored in an `int`) I'd check it with `!fix_system` here. Otherwise it looks like it was actually a numeric arg here...